resources/uk/gov/hmcts/flux/reconcile-flux-image-repository.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+
+CHART_DIRECTORY=${1}-${2}
+
+if [[ $CHART_DIRECTORY = "plum-frontend" ]]; then
+    CHART_DIRECTORY="cnp-plum-frontend"
+fi
```

Comment:
this doesn't look right, it's in https://github.com/hmcts/cnp-plum-frontend/tree/master/charts/plum-frontend

Reply:
https://github.com/hmcts/cnp-flux-config/pull/17175

Reply:
is this going to be a general issue? worth checking a few I guess